renderer/BUILD.gn

```diff
@@ -10,6 +10,7 @@ source_set("renderer") {
 
   public_deps = [
     "//chrome/renderer",
+    "//brave/content/renderer"
```

Comment:
This breaks component build.
`//brave/content/renderer` sources should be included in content not chrome.